### PR TITLE
support dispatching event on native platform

### PR DESCRIPTION
**Pull request by PPpro**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [x] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: pal/input/native/mouse-input.ts [hunk 1]
```diff
@@ -10,10 +10,24 @@ export class MouseInputSource {
...
     private _isPressed = false;
 
+    private _handleMouseDown: (mouseEvent: jsb.MouseEvent) => void;
+    private _handleMouseMove: (mouseEvent: jsb.MouseEvent) => void;
+    private _handleMouseUp: (mouseEvent: jsb.MouseEvent) => void;
+    private _boundedHandleMouseWheel: (mouseEvent: jsb.MouseWheelEvent) => void;
+
     constructor () {
+        this._handleMouseDown = this._createCallback(InputEventType.MOUSE_DOWN);
+        this._handleMouseMove = this._createCallback(InputEventType.MOUSE_MOVE);
```
Comment (dogeFu) [inline pal/input/native/mouse-input.ts:23]: createCallback里，new EventMouse 的时候，需要把外部传递进来的windowId设置给EventMouse对象

**Review by dogeFu — COMMENTED** [review]
通过对外api转发event时，需要取windowId参数

**Review by dogeFu — COMMENTED** [review]
多窗口点击没问题了，ci报错要看看

**Review by dogeFu — APPROVED** [review]

**Comment by PPpro** [comment]
自动化测试结果是已知问题

**Review by SantyWang — APPROVED** [review]

**Pull request merged**